css/partials/_page-titles.scss

```diff
@@ -21,7 +10,7 @@
 		background: none;
 		color: #111;
 		display: block;
-		padding: 1rem;
+		padding: 0 5%;
 	}
 	a.title-page {
```

Comment:
this should not be the same class as it's parent - it's inheriting all the styles (the padding is especially problematic). I know you didn't introduce this but can you change the a.title-page to .title-page-link here and in inc/content-root.php?